Automattic/jetpack
# Update reCAPTCHA constants to match Google's Verbage

Fixes #10159

#### Changes proposed in this Pull Request:

* Adds two new functions to `sharedaddy.php` to allow reCAPTCHA key to be either the legacy `RECAPTCHA_PUBLIC_KEY` values or new `RECAPTCHA_SITE_KEY` ones.
* Functions added are `sharing_recaptcha_site_key` and `sharing_recaptcha_secret_key`

#### Testing instructions:






* Go to `config.php` and define `RECAPTCHA_SITE_KEY` and `RECAPTCHA_SECRET_KEY` with values from Google. IE:
`define( 'RECAPTCHA_SITE_KEY', $YOUR_KEY );`
* Test the ReCaptcha is shown when sharing by email. 
* Update `RECAPTCHA_SITE_KEY` to legacy `RECAPTCHA_PUBLIC_KEY` and `RECAPTCHA_SECRET_KEY` to legacy `RECAPTCHA_PUBLIC_KEY` then retest.

#### Does this pull request change what data or activity we track or use?

No.

#### Proposed changelog entry for your changes:

* Update: changed ReCAPTCHA constants to match Google's verbage. Replaced `RECAPTCHA_PUBLIC_KEY` and `RECAPTCHA_PRIVATE_KEY` with `RECAPTCHA_SITE_KEY` and `RECAPTCHA_SECRET_KEY` respectively.
